supreme_court_predictions/__main__.py

```diff
@@ -49,3 +58,6 @@
 
     if args.describe_data:
         describe_data()
+
+    if args.tokenize_data:
+        tokenize_data()
```

Comment:
I think you'll need to move this above so the token data is there, should it be used in the describe function.

Reply:
gotcha